frontend/components/containers/comments/CommentSection.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <div class="font-weight-bold ml-8 mb-2">
+      Comments
+    </div>
+
+    <v-timeline
+      align-top
+      dense
+    >
+      <v-timeline-item
+        fill-dot
+        class="white--text mb-12"
+        color="green"
+        large
+      >
+        <v-textarea
+          v-model="message"
+          outlined
+          name="CommentInput"
+          label="Message"
+          value=""
+        />
+        <v-btn
+          class="white--text"
+          color="green"
+          depressed
+          :disabled="message.length === 0"
+          @click="add"
+        >
+          Send
+        </v-btn>
+      </v-timeline-item>
+      <comment v-for="comment in comments" :key="comment.id" :comment="comment" @delete-comment="remove" @update-comment="update" />
+    </v-timeline>
+  </div>
+</template>
+
+<script>
+
+import { mapActions, mapState, mapMutations } from 'vuex'
+import Comment from './Comment'
+
+export default {
+  name: 'CommentSection',
+  components: { Comment },
+
+  data() {
+    return {
+      message: ''
+    }
+  },
+  computed: {
+    ...mapState('documents', ['total', 'comments', 'current', 'selected'])
+  },
+  watch: {
+    total() {
+      this.getCommentList({
+        projectId: this.$route.params.id
+      })
+    },
+    current: {
+      handler() {
+        if (this.total !== 0) {
+          this.getCommentList({
+            projectId: this.$route.params.id
+          })
+        }
+      },
+      immediate: true
+    }
+  },
```

Comment:
what are these watchers used for?

Reply:
Resolves the problem, when reloading the annotation view. This checks if the documents are loaded, since the document ids are needed to fetch the comments.